workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -79,3 +80,75 @@ If the `delegateIp` option is used, then - if available - an IP address assigned
 |Options||
 |--|--|
 | `delegateIp` | `pod` backend only: The IP assigned to the pod will be delegated to the VM using DHCP |
+
+### `slirp` connection
+
+This connection will use the qemu user interface.
+Let the VM be exposed like a process which would be running inside a container. The requirement is that a process will bind to a port.
+
+It is a known issue that SLIRP is not recommended for anything in production.
+
+All the egress connection will came from the qemu process and all the ingress connection will go to qemu process.
+
+|Feature||
+|--|--|
+| Supported protocols | TCP/UDP only |
+
+#### Example
+this configuration allow to expose vm ports to the pod.
+```
+apiVersion: kubevirt.io/v1alpha2
```

Comment:
Wouldn't it make sense to instead include the example in cluster/examples/ (using vms-generator) and then refer to it from here? The problem with examples included in the documentation is that they tend to become obsolete / invalid, since we don't validate their correctness in any way. If examples are vms-generator based then they are validated by admission controller etc.